lms/djangoapps/grades/api/views.py

```diff
@@ -0,0 +1,139 @@
+""" API v0 views. """
+import logging
+
+from django.http import Http404
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from rest_framework import status
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from lms.djangoapps.ccx.utils import prep_course_for_grading
+from lms.djangoapps.courseware import courses
+from lms.djangoapps.grades.new.course_grade import CourseGradeFactory
+from openedx.core.lib.api.authentication import OAuth2AuthenticationAllowInactiveUser
+from openedx.core.lib.api.view_utils import DeveloperErrorViewMixin
+
+log = logging.getLogger(__name__)
+
+
+class UserGradeView(DeveloperErrorViewMixin, GenericAPIView):
+    """
+    **Use Case**
+
+        * Get the current course grades for users in a course.
+          Currently, getting the grade for only an individual user is supported.
+
+    **Example Request**
+
+        GET /api/grades/v0/course_grade/{course_id}/users/?username={username}
+
+    **GET Parameters**
+
+        A GET request must include the following parameters.
+
+        * course_id: A string representation of a Course ID.
+        * username: A string representation of a user's username.
+
+    **GET Response Values**
+
+        If the request for information about the course grade
+        is successful, an HTTP 200 "OK" response is returned.
+
+        The HTTP 200 response has the following values.
+
+        * username: A string representation of a user's username passed in the request.
+
+        * course_id: A string representation of a Course ID.
+
+        * passed: Boolean representing whether the course has been
+                  passed according the course's grading policy.
+
+        * percent: A float representing the overall grade for the course
+
+        * letter_grade: A letter grade as defined in grading_policy (e.g. 'A' 'B' 'C' for 6.002x) or None
```

Comment:
I am not sure what it means: I copied the code (and description) from the `courseware` apis.
From the code, I think it means that the grading policy does not define a `letter_grade` for the `percentage` associated.
Should I add this?